src/v0/util/index.js

```diff
@@ -1746,6 +1746,13 @@ const validatePhoneWithCountryCode = (phone) => {
   const regex = /^\+(?:[\d{] ?){6,14}\d$/;
   return !!regex.test(phone);
 };
+const isEmail = inputString => {
```

Comment:
I am seeing validateEmail has same functionality as this. We should make use of the same function. If needed, rename the function wherever it is used.
We can do the renaming if the number of references of `validateEmail` are less.

Reply:
As validateEmail and isEmail has two different meanings as explained by @koladilip but has same regex testing I have created a new `function` as is email and redirected it to `validateEmail`

Reply:
We are tracking this in this [conversation](https://github.com/rudderlabs/rudder-transformer/pull/2404#discussion_r1274705852)